k8s/clusters/eu-central-1a/out/terraform/kubernetes.tf

```diff
@@ -0,0 +1,483 @@
+  type              = "ingress"
+  security_group_id = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+}
+
+resource "aws_security_group_rule" "master-egress" {
+  type              = "egress"
+  security_group_id = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port         = 0
+  to_port           = 0
+  protocol          = "-1"
+  cidr_blocks       = ["0.0.0.0/0"]
+}
+
+resource "aws_security_group_rule" "node-egress" {
+  type              = "egress"
+  security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port         = 0
+  to_port           = 0
+  protocol          = "-1"
+  cidr_blocks       = ["0.0.0.0/0"]
+}
+
+resource "aws_security_group_rule" "node-to-master-protocol-ipip" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 0
+  to_port                  = 65535
+  protocol                 = "4"
+}
+
+resource "aws_security_group_rule" "node-to-master-tcp-1-2379" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 1
+  to_port                  = 2379
+  protocol                 = "tcp"
+}
+
+resource "aws_security_group_rule" "node-to-master-tcp-2382-4001" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 2382
+  to_port                  = 4001
+  protocol                 = "tcp"
+}
+
+resource "aws_security_group_rule" "node-to-master-tcp-4003-65535" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 4003
+  to_port                  = 65535
+  protocol                 = "tcp"
+}
+
+resource "aws_security_group_rule" "node-to-master-udp-1-65535" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 1
+  to_port                  = 65535
+  protocol                 = "udp"
+}
+
+resource "aws_security_group_rule" "ssh-external-to-master-0-0-0-0--0" {
```

Comment:
we block external access

Reply:
We setup a nacl for this, its not automated in any way but we have a nacl to block external access